wagtail/embeds/oembed_providers.py

```diff
@@ -522,8 +522,8 @@
 spotify = {
     "endpoint": "https://open.spotify.com/oembed",
     "urls": [
-        r"^https?://open\.spotify\.com/.+$",
-        r"^https?://spoti\.fi/.+$",
+        r"^https://open\.spotify\.com/.*$",
+        r"^spotify:.*$"
```

Comment:
I'm nor sure we would want the Spotify scheme here, I would assume most URLs users copy would not be this format. 

Do you have an example of docs link where this would be the case?

Reply:
@lb- Thank you for pointing that out. You are correct; Spotify scheme is not widely used by most users. Therefore, I have removed it. I appreciate your help in correcting my mistake.